core/controllers/base.py

```diff
@@ -86,7 +86,7 @@
     },
     'Additional links to show in the sidebar menu.',
     default_value=[{
-        'name': 'Site Feedback',
+        'name': 'I18N_SIDEBAR_ADDITIONAL_LINK_SITE_FEEDBACK',  # Site Feedback
```

Comment:
Actually let's leave this one out for now -- it's going to be a bit of a pain because this is a config property modifiable by admins. We'll have to figure it out later. (An upcoming UI redesign may actually get rid of this property altogether, so that's another good reason to delay.)